From a1bd733809ff01c64a8a304a45e57277a5a98463 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 8 Dec 2023 19:35:37 +0100
Subject: [PATCH] meson: fix installation of ukify

ln_s was added in upstream later on. It's not present in this branch.
Fixup for b98da2d9e7.

Related: RHEL-13199
---
 meson.build | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index 936e612a01..e09c426a72 100644
--- a/meson.build
+++ b/meson.build
@@ -3868,9 +3868,9 @@ ukify = custom_target(
 if want_ukify
         public_programs += ukify
 
-        meson.add_install_script(sh, '-c',
-                                 ln_s.format(bindir / 'ukify',
-                                             rootlibexecdir / 'ukify'))
+        meson.add_install_script(meson_make_symlink,
+                                 bindir / 'ukify',
+                                 rootlibexecdir / 'ukify')
 endif
 
 if want_tests != 'false' and want_kernel_install
